lib/order.rb

```diff
@@ -0,0 +1,66 @@
+class Order 
+  attr_reader :basket, :selected_dish, :menu, :total_price
+
+  def initialize
+    @menu = [
+      { name: "Bangers & Mash", price: 8.49, available: true },
+      { name: "Chips & Gravy", price: 3.19, available: true },
+      { name: "Fish Pie", price: 9.89, available: true },
+      { name: "Casserole", price: 7.79, available: true }
+    ]
+    @basket = []
+    @selected_dish = nil
+    @total_price = 0
+  end
+  
+  def see_list
+    @menu.each_with_index { |item, index|
+      puts "#{index + 1}. #{item[:name]} - £#{item[:price]}"
+    }
+  end
+
+  def find_dish_details(dish)
+    # [0] had to select the 0th index because the hash is in an array
+    @selected_dish = @menu.select { |item| 
+      item[:name] == dish
+    }[0]
+  end
+
```

Comment:
Some of these methods correspond to attributes you might associate with a Menu rather than the actual order, and could maybe be moved over to a Menu class - ie. find_dish_details, make_unavailable